### DISCUSSION TITLE
Verification Code

### DISCUSSION THREAD

**Discussion by Khoi1201** [vercel/community#5025] [Account Issue] [answered]
### Question

Hi, I can't get a verification code with my phone number, I have tried using another phone number and still can't get it, and now my number is blocked. I'm using github account to connect vercel and it require a phone verification.

![vercel](https://github.com/vercel/community/assets/90588753/916e5c83-47de-427e-a6b9-9ec90f90fca6)

**Comment by swarnava** [comment, marked as answer]
Hi, Pleas email us at registration@vercel.com from your registered email. Thank you!

**Comment by Khoi1201** [comment]
Hi, I have sent my email but still did not get any support, can you check
it again please?

Vào Th 6, 1 thg 12, 2023 vào lúc 02:04 Swarnava Sengupta <
***@***.***> đã viết:
